### DISCUSSION TITLE
Start mapviewer with country boundaries included

### DISCUSSION THREAD

**Discussion by 1969bahati** [lsc-hubs/kenya-hub#27] [General]
Kenya county shapefiles should pop up first before the data is displayed

**Comment by pvgenuchten** [comment]
I like this suggestion mr @1969bahati, country boundaries (currently labeled adm3) can easily be activated in the default map, but i agree they could get more priority, maybe even display by default

![image](https://github.com/lsc-hubs/kenya-hub/assets/299829/39a04211-ba2c-4f4e-8201-4d1ece3ef049)
